hack/release.sh

```diff
@@ -397,4 +397,15 @@ main
 echo
 echo
 echo "Release complete; see $(s3_url)"
+echo "Use the following text to announce the release:"
+echo
+echo "We have just pushed $VERSION to test.docker.com. You can download it with the following:"
+echo
+echo "Ubuntu/Debian: curl -sSL https://test.docker.com/ubuntu | sh or curl -sSL https://test.docker.com/ | sh"
```

Comment:
> It wasn't supposed to be a script you actually run, we just didn't want to
write HTML to describe the couple simple steps to actually install.

Oh, LOL!  In that case, perhaps you can comment on https://github.com/docker/docker/pull/14343